reducer/src/main/java/com/graphicsfuzz/reducer/ReductionDriver.java

```diff
@@ -302,6 +304,7 @@ public String doReduction(
             addInitializers,
             currentShaderJobShortName);
         passManager.notifyInteresting(interesting);
+
```

Comment:
Please avoid adding redundant whitespace: do a git diff with the last commit before your changes and get rid of any whitespace changes that are not part of the PR.

(Sometimes we do PRs whose intent is to clean up formatting, but it's best to do them separately.)